src/lib/Controls.js

```diff
@@ -175,9 +175,15 @@ class Controls {
      * @param {Event} event - A DOM-normalized event object.
      * @return {void}
      */
-    clickHandler = () => {
+    clickHandler = event => {
+        if (event) {
+            event.stopPropagation();
+        }
+
+        if (this.hasTouch) {
```

Comment:
Do we want to hide the controls only for touch-enabled devices?

Reply:
Yes, I believe that was the intention of the original code as it didn’t register the click handler unless it was a touch device. Now that we are using it, probably should keep the same intention, while only hiding on mobile devices.